server.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"app/middleware/auth"
```

Comment:
It is working now, the thing is that the docker image we use puts the code under `...src/app` so every package is in the app namespace. We should normalize this so it works in both local and docker environment